Sources/RegexBenchmark/Suite/CssRegex.swift

```diff
@@ -6,7 +6,7 @@ extension BenchmarkRunner {
     let r = #"--([a-zA-Z0-9_-]+)\s*:\s*(.*?);"#
 
     let css = CrossBenchmark(
-      baseName: "css", regex: r, input: Inputs.swiftOrgCSS)
+      baseName: "Css", regex: r, input: Inputs.swiftOrgCSS)
```

Comment:
FWIW, I believe Swift's naming convention would prefer to keep them all capitalized or all lower case. Not a big deal to me.